releasenotes/notes/improve-performance-clifford-dot-method-299df21a64408045.yaml

```
@@ -0,0 +1,9 @@
+---
+features_quantum_info:
+  - |
+    Improve the performance of :meth:`~qiskit.quantum_info.Clifford.dot`,
+    when it is composed with a :class:`QuantumCircuit`.
+    In this case, we use new internal methods to iteratively update the Clifford
+    by prepending with standard Clifford gates.
```

Do we need to stress that the new methods are `internal`?
```suggestion
    In this case, we iteratively update the Clifford by prepending it with standard Clifford gates.
```

---

done